src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs

```diff
@@ -2,36 +2,36 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
+    public class StageIdFilterServiceAttribute : ServiceFilterAttribute

```

Comment:
Can this be renamed to, e.g. `EnsureStageExistsAttribute` or something? Makes it easier to grasp what it does from reading the controller.